### DISCUSSION TITLE
Syntax error when running jobrunr using oracle version before 12 (ex: 11g)

### DISCUSSION THREAD

**Discussion by thupq** [jobrunr/jobrunr#923] [General]
In class org.jobrunr.storage.sql.common.BackgroundJobServerTable with method: getLongestRunningBackgroundJobServerId (method to get serverId)
generate and execute the command: "select id from jobrunr_backgroundjobservers ORDER BY firstHeartbeat ASC OFFSET ? ROWS FETCH NEXT ? ROWS ONLY" to get the serverId.
However, this syntax is only supported from version 12c release as shown in the following oracle documentation: https://www.oracletutorial.com/oracle-basics/oracle-fetch/

**Comment by rdehuyss** [comment]
Oracle 11g was released in 2007 and is already end of life for 3 years 🤣.

Welcome to 2024!
